packages/runtime/datastore/src/fluidHandle.ts

```diff
@@ -24,6 +21,22 @@ export class FluidObjectHandle<T extends FluidObject = IFluidObject> implements
         return this.routeContext.isAttached;
     }
 
+    /**
+     * Tells whether the object of this handle is visible in the container.
+     */
+    private get visible(): boolean {
+        /**
+         * If the object of this handle is attached, it is visible in the container. This is a work around the scenario
+         * where the object becomes visible but the handle does not get this notification. For example, handles to a DDS
+         * other than the default handle won't know if the DDS becomes visible after the handle was created.
+         */
+        if (this.isAttached) {
+            this._visible = true;
```

Comment:
could we use the this.visibilityState === locallyVisible || this.visibilityState === globallyVisible instead of a flag here?

Reply:
I don't want to add visibility state here because it doesn't need it. Why add extra state if its not required?

Reply:
I just realized this was the handle